modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase

```diff
@@ -1106,13 +1106,11 @@ definition {
 					locator1 = "CreateObject#SELECT_ENTRY_IN_RELATIONSHIP_TAB");
 			}
 
-			ClickNoError(
-				key_titleFieldName = "Entry Test C",
-				locator1 = "CreateObject#SELECT_ENTRY_IN_RELATIONSHIP_TAB");
+			SelectFrame(value1 = "relative=top");
 
-			WaitForElementPresent(
-				key_fieldLabel = "Entry Test C",
-				locator1 = "ObjectAdmin#VIEW_OBJECT_FIELD_LABEL");
+			Click.clickNoMouseOver(locator1 = "Modal#CLOSE_BUTTON");
+
+			CreateObject.addRelationshipAndSelectEntry(titleFieldName = "Entry Test C");
```

Comment:
@thaiscabrals hmm I don't think we need to repeat the same steps to make sure `Entry Test C` is there. Would just selecting the frame and then waiting for the page to load work just like in `CreateObject.addRelationshipAndSelectEntry`?